kits/agp1/contracts/AGP1Kit.sol

```diff
@@ -0,0 +1,86 @@
+pragma solidity 0.4.24;
+
+import "@aragon/apps-finance/contracts/Finance.sol";
+import "@aragon/apps-vault/contracts/Vault.sol";
+import "@aragon/apps-voting/contracts/Voting.sol";
+import "@aragon/apps-shared-minime/contracts/MiniMeToken.sol";
+
+import "@aragon/os/contracts/apm/APMNamehash.sol";
+import "@aragon/os/contracts/common/IsContract.sol";
+
+import "@aragon/kits-base/contracts/KitBase.sol";
+
+
+contract AGP1Kit is KitBase, APMNamehash, IsContract {
+
+    uint64 constant public MAIN_VOTING_SUPPORT = 50 * 10**16; // > 50%
+    uint64 constant public MAIN_VOTING_QUORUM = 0; // Just 1 vote is enough
+    uint64 constant public MAIN_VOTING_VOTE_TIME = 48 hours;
+
+    uint64 constant public META_TRACK_VOTING_SUPPORT = 666666666666666667; // > two thirds
```

Comment:
Tiny detail: Ending in a 6 is the strict way of requiring > 2/3, as `666666666666666667` is already > 2/3 and the voting app has strict thresholds.